.github/workflows/sql-workbench-release-workflow.yml

```diff
@@ -7,8 +7,8 @@ on:
 
 env: 
   PLUGIN_NAME: query-workbench-dashboards
-  OPENSEARCH_VERSION: '1.2'
-  OPENSEARCH_PLUGIN_VERSION: 1.2.0.0
+  OPENSEARCH_VERSION: '1.3'
```

Comment:
looks like dashboards `1.3` branch is not cut yet, to unblock CI you can change it to `1.x`